src/css/custom.css

```diff
@@ -2,8 +2,22 @@
   --ifm-light-background-color: var(--ifm-color-emphasis-100);
 }
 
+html[data-theme='light'] {
+  --ifm-color-primary: #1554B7;
+  --ifm-link-color: #1554B7;
+
+  .DocSearch-Button-Placeholder {
+    color: #494B5F;
```

Comment:
I think this can be set using the `--docsearch-muted-color` css variable.
This makes it easier to maintain when we upgrade docusaurus.

Reply:
Good call, addressed in latest commit